src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/IntermediateTokenBaseline.txt

```diff
@@ -0,0 +1,82 @@
+Mvc/PagesPropertyInjectionPass.cs:32:34
+Mvc/PagesPropertyInjectionPass.cs:40:36
+Language/Intermediate/IntermediateToken.cs:22:103
+Language/Components/ComponentBindLoweringPass.cs:404:22
+Language/Components/ComponentBindLoweringPass.cs:421:23
```

Comment:
Having the line and column number is going to get really annoying when editing files if you have to edit this baseline file by hand. 

Do we:
 
A) Just track the filename
B) Add a fixer a la shipped APIs that allows you to automatically update the baseline?

I lean towards b, because I think its valuable to have it tracked all the places its done, but don't have a super strong preference.

Reply:
Could we track just the file name and text content of the line instead?